pkg/cmd/pipeline/translate.go

```diff
@@ -136,6 +136,26 @@ func UpdateConfigMap(ctx context.Context, cmap *apiv1.ConfigMap, data *CfgData,
 	return configmaps.Deploy(ctx, cmap, cmap.Namespace, c)
 }
 
+// UpdateConfigMap updates the configmaps fields
+func UpdateEnvs(ctx context.Context, name, namespace string, envs []string, c kubernetes.Interface) error {
```

Comment:
Can we add a test for this func

Reply:
done cb978af6de11e2423a14ec3d7cb70306b6ec7e9c